tests/framework/config/cluster.go

```diff
@@ -27,4 +27,6 @@ type ClusterConfig struct {
 	PeerTLS           TLSConfig
 	ClientTLS         TLSConfig
 	QuotaBackendBytes int64
+	AuthToken         string
+	SnapshotCount     uint64
```

Comment:
`SnapshotCount` unrelated change?

Reply:
This field will be used in real auth tests to trigger frequent snapshot to validate consistency.